server/routes/bookings/bookingsUtils.test.ts

```diff
@@ -12,57 +14,69 @@ describe('Bookings utils', () => {
         ['VISITOR_CANCELLED', 'This visit was cancelled by a visitor.'],
         ['fallback-any-other-status', 'This visit was cancelled by the prison.'],
       ])('CANCELLED visit with outcome status %s', (outcomeStatus: VisitDetails['outcomeStatus'], text: string) => {
-        const visit = TestData.visitDetails({ visitStatus: 'CANCELLED', outcomeStatus })
+        const visit = TestData.visitDetails({ visitStatus: 'CANCELLED', visitSubStatus: 'CANCELLED', outcomeStatus })
         const expectedMessage: MoJAlert = {
           variant: 'information',
           title: 'Visit cancelled',
           showTitleAsHeading: true,
           text,
         }
 
-        expect(getVisitMessages(visit, 'Hewell (HMP)')).toStrictEqual([expectedMessage])
+        expect(getVisitMessages({ visit, prisonName })).toStrictEqual([expectedMessage])
       })
     })
 
-    it('Requested visit', () => {
+    it('visit request', () => {
       const visit = TestData.visitDetails({ visitSubStatus: 'REQUESTED' })
       const expectedMessage: MoJAlert = {
         variant: 'information',
         title: 'Your request needs to be reviewed',
         showTitleAsHeading: true,
-        text: 'This visit is not booked yet. It needs to be checked by Hewell (HMP).',
+        text: `This visit is not booked yet. It needs to be checked by ${prisonName}.`,
       }
 
-      expect(getVisitMessages(visit, 'Hewell (HMP)')).toStrictEqual([expectedMessage])
+      expect(getVisitMessages({ visit, prisonName })).toStrictEqual([expectedMessage])
     })
 
-    it('Rejected visit', () => {
-      const visit = TestData.visitDetails({ visitStatus: 'REJECTED', visitSubStatus: 'REJECTED' })
+    it('rejected visit request', () => {
+      const visit = TestData.visitDetails({ visitStatus: 'CANCELLED', visitSubStatus: 'REJECTED' })
```

Comment:
Some of these tests had invalid values (e.g. `REJECTED` for `visitStatus`). TypeScript didn't complain because there version of the API definition was incorrect; I've updated it in this PR. `visitStatus` can only be `BOOKED | CANCELLED`.